tests/roles/dataplane_adoption/tasks/main.yaml

```diff
@@ -258,16 +258,16 @@
         - configure-network
         - validate-network
         - install-os
+        - ceph-hci-pre
```

Comment:
@fmount Is it ok to move ceph-hci-pre here? It matches where it is in this example:

https://github.com/openstack-k8s-operators/openstack-operator/blob/main/config/samples/dataplane/pre_ceph_hci/values.yaml#L145

Reply:
+1 from me.
Besides, that role is supposed to generate and apply firewall rules in preparation to Ceph deploy [1], so I think it's ok to have it after `install-os`.

[1] https://github.com/openstack-k8s-operators/edpm-ansible/blob/main/roles/edpm_ceph_hci_pre/tasks/firewall.yml